doc/managing_wallet.md

```diff
@@ -0,0 +1,115 @@
+
+The Bitcoin Core wallet was originally a collection of unrelated private keys with their associated addresses. If a non-HD wallet had generated a key/address, had given that address out and then had restored a backup from before that key's generation, then any funds sent to that address would have been lost definitively.
+
+However, [version 0.13](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.13.0.md) introduced HD wallets. Users no longer lose funds when restoring old backups, as long as the addresses used were from the wallet's HD seed (since all private keys can be derived from the seed).
+
+So, theoretically, a single backup is enough. But it is recommended to make regular backups (1 time a day or a week) or when there is a relevant amount of new transactions in the wallet.
+
+### 1.5 Restoring The Wallet From a Backup
+
+To restore a wallet, an empty one must be created  first.  The backup file then rewrites the `wallet.dat` of this new wallet.
+
+The user must unload the wallet and load it again after replacing the file.
+
+```
+$ bitcoin-cli createwallet "restored_wallet"
+$ bitcoin-cli unloadwallet "restored_wallet"
+$ cp ~/Backups/backup01.dat ~/.bitcoin/wallets/restored_wallet/wallet.dat
+$ bitcoin-cli loadwallet "restored_wallet"
+```
+
+After these steps, the wallet balance can be checked.
+
+`$ bitcoin-cli -rpcwallet="restored_wallet" getbalance`
+
+### 1.6 `-rescan` and `-reindex`
+
+The `-rescan` argument rescans the blockchain for missing wallet transactions during startup.
+
+`$ bitcoind -rescan`
+
+This is only necessary if there are missing transactions after restoring the wallet.
+
+A pruned node, however, is incompatible with the `-rescan` option, since it does not have the data to check for relevant transactions.
+
+In that case, it is necessary to restart the node with the `-reindex` command-line option to start over with the initial sync. The wallet will scan for relevant transactions during the synchronization and rediscover the funds and transaction history of the wallet.
+
+`$ bitcoind -reindex`
+
+### 1.7 Dumping The Wallet
+
+Alternatively, the `dumpwallet` command can be used to backup the wallet. It dumps all wallet keys (including the master private key) in a human-readable format into a file.
+
+Note that this file is not encrypted and the keys are exposed. An attacker in possession of the file could recreate the wallet and gain access to the keys.
+
+The file is generated on the server-side and so the user must have access to the server folders.
+
+`$ bitcoin-cli -rpcwallet="wallet-01" dumpwallet /home/node01/Backups/dump01.txt`
+
+### 1.8 Importing The Wallet From a Dump File
+
+The command `importwallet` imports keys from a wallet dump file.
+
+The first step is to create a new wallet.
+
+`$ bitcoin-cli createwallet "from_dump_file"`
+
+Then this command can be called.
+
+`$ bitcoin-cli -rpcwallet="from_dump_file" importwallet /home/node01/Backups/dump01.txt`
+
```

Comment:
```suggestion

> :information_source: **NOTE:** Importing wallets is disabled when blocks are pruned
```

Reply:
Removed content about dump file as suggested in https://github.com/bitcoin/bitcoin/pull/22523#discussion_r674488255